dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/response/FacetCategory.java

```diff
@@ -1,11 +1,12 @@
 package edu.harvard.iq.dataverse.search.response;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class FacetCategory {
 
     private String name;
-    private List<FacetLabel> facetLabel;
+    private List<FacetLabel> facetLabel = new ArrayList<>(); // TODO: should be plural
```

Comment:
Did you forget about it or is it more complicated than just a simple refactor?

Reply:
Actually, I noticed that at the last moment. I will correct this